lib/ReactViews/Custom/registerCustomComponentTypes.js

```diff
@@ -148,6 +150,15 @@ const registerCustomComponentTypes = function(terria) {
         }
         const url = defined(sources) ? sources[0] : undefined;
         const tableStructure = tableStructureFromStringData(getSourceData(node, children));
+
+        const colors = splitStringIfDefined(node.attribs['colors']);
+        if (defined(colors)) {
+            const getColorCallback = function(columnId) {
+                return colors[columnId];
+            };
+            tableStructure.getColorCallback = getColorCallback;
```

Comment:
Minor suggestion - the new functional format would be more succinct - eg. `tableStructure.getColorCallback = index => colors[index];`.